ppr-ui/src/components/registration/length-trust/EditTrustIndenture.vue

```diff
@@ -0,0 +1,132 @@
+<template>
```

Comment:
Please add a test suite for this that simulates the user updating trust indenture / pressing 'done' and 'cancel' and verifies the store updated correctly / didn't update if they pressed cancel + that it emits correctly in each case

Reply:
Done.